Source/Public/Update-VSTeamGitRepositoryDefaultBranch.ps1

```diff
@@ -0,0 +1,61 @@
+# Sets the default branch for a Git repository in your Azure DevOps or Team Foundation Server account.
+#
+# Get-VSTeamOption 'git' 'repositories'
+# id              : 225f7195-f9c7-4d14-ab28-a83f7ff77e1f
+# area            : git
+# resourceName    : repositories
+# routeTemplate   : {project}/_apis/{area}/{resource}/{repositoryId}
+# http://bit.ly/Add-VSTeamGitRepository
+function Update-VSTeamGitRepositoryDefaultBranch {
+   [CmdletBinding()]
+   param(
+      [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [string] $Name,
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [vsteam_lib.ProjectValidateAttribute($false)]
+      [ArgumentCompleter([vsteam_lib.ProjectCompleter])]
+      [string] $ProjectName,
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [string] $DefaultBranch
+   )
+   begin {
+      try {
+         # $Repo = Get-VSTeamGitRepository -Name $Name -ProjectName $ProjectName
+         $Repo = _callAPI -Method Get -ProjectName $ProjectName `
+            -Area "git" `
+            -Resource "repositories" `
+            -id $Name `
+            -Version $(_getApiVersion Git)
+      } catch {
+         Write-Warning "A repo named $Name could not be found in the project $ProjectName..."
+         throw $PSItem.Exception.Message
+      }
+      if ($DefaultBranch -notlike "*refs/head*") {
+         $DefaultBranch = 'refs/head/{0}' -f $DefaultBranch
+      }
+   }
+   process {
+      
+      
+      $body = @{  
+         # name          = $Name
```

Comment:
Also remove as soon as it is clear what you wanna do.

Reply:
Cleaning up, and re-committing.